### PR TITLE
Issue #62, Issue #71, Issue #89, cleanup.

### PR DESCRIPTION
**Pull request by alegag**
Issue #89: Added methods show() and hide() in addition to toggle().

Issue #71: Added an optional autoClose=true to close the dialog on date/time selection.

Issue #62: Option for 24-hour in an inner-circle.

Removed "picker" variable definied and not used.

Removed "start" variable definied and not used.

Removed "spaces" mixed with "tabs".

### CODE REVIEW

**Comment by alegag** [comment]
@puranjayjain Sorry, I saw the guideline after the pull request.  I wanted to make multiple commit at first, should have !  Next time !

File: src/js/mdDateTimePicker.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ class mdDateTimePicker {
 		this._ok = ok
 		this._cancel = cancel
 		this._colon = colon
+		this._auto_close = auto_close
```
Comment (alegag) [inline src/js/mdDateTimePicker.js]: Fixed camelCase is last commit.

File: dist/js/mdDateTimePicker.js [hunk 2]
```diff
@@ -97,6 +100,7 @@
 			this._ok = ok;
 			this._cancel = cancel;
 			this._colon = colon;
+			this._auto_close = auto_close;
```
Comment (alegag) [inline dist/js/mdDateTimePicker.js]: Fixed camelCase is last commit.

File: dist/js/mdDateTimePicker.js [hunk 3]
```diff
@@ -83,7 +84,9 @@
 			    _ref$cancel = _ref.cancel,
 			    cancel = _ref$cancel === undefined ? 'cancel' : _ref$cancel,
 			    _ref$colon = _ref.colon,
-			    colon = _ref$colon === undefined ? !0 : _ref$colon;
+			    colon = _ref$colon === undefined ? !0 : _ref$colon,
+			    _ref$auto_close = _ref.auto_close,
+			    auto_close = _ref$auto_close === undefined ? false : _ref$auto_close;
```
Comment (alegag) [inline dist/js/mdDateTimePicker.js]: Fixed camelCase is last commit.

**Comment by puranjayjain** [comment]
@alegag good work you can do multiple commits still I'll keep this open until you do no problem. You can do the multiple commits and in the end squash them together in one. I'll help you if you get stuck don't worry.

**Comment by alegag** [comment]
@puranjayjain Here you go ! As three different commits for the two issues and the little cleanup.

**Comment by alegag** [comment]
@puranjayjain Okay, last commit I was pulling for now.  That commit cover issue #62 as an optional feature (needs to set inner24=true to use it).

That commit touch the SCSS.  I did not regenerated all the themes as I'm not familiar with the procedure.

I'll let your review all of this now.

**Comment by puranjayjain** [comment]
ok @alegag I'll do it

**Comment by puranjayjain** [comment]
Nice work @alegag thank you merging it now.

**Pull request merged**